### PR TITLE
cbflib: new package

### PR DESCRIPTION
**Pull request by ketsubouchi**
dependency package of openrasmol #21530

### CODE REVIEW

File: var/spack/repos/builtin/packages/cbflib/package.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    version('0.9.2', sha256='367e37e1908a65d5472e921150291332823a751206804866e752b793bca17afc')
+
+    depends_on('m4', type='build')
+
+    patch('cbf_f16.patch')
+    patch('cbf_int.patch')
+
+    def setup_build_environment(self, env):
+        ce = Executable(self.compiler.cc)
+        ce('-E', join_path(os.path.dirname(__file__), "checkint.c"),
```
Comment (haampie) [inline var/spack/repos/builtin/packages/cbflib/package.py:27]: Can this be upstreamed to cbflib?

**Review by tldahlgren — COMMENTED** [review]
FWIW Confirmed the specified version is available at the download site.

**Comment by ketsubouchi** [comment]
I created Issue in their github.
https://github.com/yayahjb/cbflib/issues/24
(old version is needed and not available in github)

**Comment by ketsubouchi** [comment]
I rerun tests, results seem OK.

**Review by alalazo — APPROVED** [review]

**Pull request merged**